src/analysis/convex/body.lean

```diff
@@ -7,7 +7,9 @@ import analysis.convex.basic
 import analysis.normed_space.basic
 import data.real.nnreal
 import data.set.pointwise.basic
+import topology.metric_space.metrizable
 import topology.subset_properties
+import topology.metric_space.hausdorff_distance
```

Comment:
Have you looked at what this does to the import graph? If it's bad, or you haven't looked, perhaps this can go in a new file?

Reply:
This file is already a leaf file.

Reply:
Okay!